features/polyExplorer/src/screens/stories/digitalGiantsStory.jsx

```diff
@@ -1,87 +1,57 @@
-import React from "react";
+import React, { useContext } from "react";
 
 import ClusterStory from "../../components/clusterStory/clusterStory.jsx";
-//import { ExplorerContext } from "../../context/explorer-context.jsx";
+import { ExplorerContext } from "../../context/explorer-context.jsx";
 import i18n from "../../i18n.js";
-import SectionTitle from "../../components/clusterStories/sectionTitle.jsx";
-import Tab from "../../components/clusterStories/tab.jsx";
+//import SectionTitle from "../../components/clusterStories/sectionTitle.jsx";
```

Comment:
You know that... never comment out code. Personal pet peeve, I know, so take it with a grain of salt.

Reply:
I agree, yet they are only deactivated to please the linter. They will be brought back within days, like as soon as the first section is added :D